app/src/hooks/useLookupName.tsx

```diff
@@ -1,17 +1,24 @@
+import { useCallback, useEffect, useState } from 'react'
+
 import { Network } from '@/models/chain'
 import { lookupName } from '@/utils/transfer'
-import { useCallback, useEffect, useState } from 'react'
 
-const useLookupName = (network: Network, address: string) => {
-  const [accountName, setAccountName] = useState<string | null>()
+const useLookupName = (network: Network | null, address: string | null) => {
```

Comment:
Can we also do this?
```suggestion
const useLookupName = (network?: Network, address?: string) => {
```